Formula/stubby.rb

```diff
@@ -0,0 +1,84 @@
+class Stubby < Formula
+  desc "DNS privacy enabled stub resolver service based on getdns"
+  homepage "https://getdnsapi.net/blog/dns-privacy-daemon-stubby/"
+  url "https://getdnsapi.net/releases/getdns-1-1-1/getdns-1.1.1.tar.gz"
+  sha256 "fa414c30d5f2d2b2453b5cec77362b4cc0f44d440be5893233748d82bd6a1a56"
+
+  head do
+    url "https://github.com/getdnsapi/getdns.git", :branch => "develop"
+
+    depends_on "libtool" => :build
```

Comment:
Within a given category, yes.

build before required, required before recommended, recommended before optional, and then within each category alphabetized